src/endpoint/sts/ops/sts_post_assume_role.js

```diff
@@ -35,7 +35,7 @@ async function assume_role(req) {
                 Credentials: {
                     AccessKeyId: access_keys.access_key.unwrap(),
                     SecretAccessKey: access_keys.secret_key.unwrap(),
-                    Expiration: expiry,
+                    Expiration: `in ${expiry.toString()}ms`,
```

Comment:
what is this expiration format? from docs I see it should be specific time, and not this duration.